.github/dependabot.yml

```diff
@@ -35,6 +35,8 @@ updates:
 
     ignore:
       - dependency-name: '@wireapp/avs'
+      # Wait until we have next release with wasm folder
+      - dependency-name: '@mediapipe/tasks-vision'
```

Comment:
maybe we could specify only 0.10.16?
Let me look if it's doable

Reply:
Yes, it doable. 
The plan is to remove it once we have new release.
As it is not a long term solution, I didn't mention version.

Reply:
```suggestion
      - dependency-name: '@mediapipe/tasks-vision'
        versions:
          - '0.10.16'
```

Reply:
> Yes, it's doable. The plan is to remove it once we have new release. As it is not a long term solution, I didn't mention version.

Kinda my reasoning for specifying the version, it would bump to the next version and either break or not.
Relying on us manually checking a single npm package for update sounds very liable to be forgoten about :sweat_smile:

Reply:
I am not sure if there are multiple ways to do it. As per documentation:
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#ignore-updates-for-a-specific-version